### PR TITLE
AISDK-205: Add Language Parameter to the Streaming Client

### PR DESCRIPTION
**Pull request by amikofalvy**

### CODE REVIEW

File: src/streaming-client.ts [hunk 1]
```diff
@@ -104,6 +104,9 @@ export class RevAiStreamingClient extends EventEmitter {
             if (config.transcriber !== null && config.transcriber !== undefined) {
                 url += `&transcriber=${encodeURIComponent(config.transcriber)}`;
             }
+            if (config.language !== null && config.language !== undefined) {
```
Comment (dmtrrk) [inline src/streaming-client.ts]:
```suggestion
            if (config.language) {
```

Comment (amikofalvy) [reply src/streaming-client.ts]: Do you think we should update all of the conditionals?

**Review by dmtrrk — APPROVED** [review]
minor proposal

File: src/streaming-client.ts (hunk 1 above)
Comment (dmtrrk) [reply src/streaming-client.ts]: this might be a techdebt, so I approved this PR.
if it is non-trivial change, let's have a separate tech debt story

Comment (amikofalvy) [reply src/streaming-client.ts]: It doesn't work for the numerical parameters such as delete_after_seconds because `0` is a `falsey` value.

Comment (dmtrrk) [reply src/streaming-client.ts]: it works for language and everything what is not `0` - for any string values actually (except `"0"` string).

Comment (amikofalvy) [reply src/streaming-client.ts]: Yes. I have updated the conditionals to reflect this. Please check again.

Comment (dmtrrk) [reply src/streaming-client.ts]: looks great

**Pull request merged**